public/less/themes/arc-green.less

```diff
@@ -85,6 +85,59 @@ body {
     color: #9e9e9e;
 }
 
+/* firefox scroll bars */
+
+html,
+body,
+* {
+    scrollbar-width: thin;
+    scrollbar-color: #87ab63 rgba(255,255,255,0.1);
+}
+
+/* webkit scrollbars */
+
+html::-webkit-scrollbar,
+body::-webkit-scrollbar,
+html ::-webkit-scrollbar,
+body ::-webkit-scrollbar {
```

Comment:
I think you can replace this and similar selector groups with a single `::-webkit-scrollbar` selector.

Reply:
you right this could be done with less css code.
i had to keep at least 2 lines:
`body ::-webkit-scrollbar` as is overwritten by semantic.min.css otherwise
`::-webkit-scrollbar` to also apply on `html` tag

change done

Reply:
I'd prefer if you use `!important` in that case. It makes the intention much more clear that way.